### PR TITLE
framed.doc: Relayout the post message test harness

### PR DESCRIPTION
**Pull request by hfiguiere**
Change-Id: Idbd1b5d89ea53d4f9925b727e8db2854ac23b17f

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [x] Code is properly formatted
- [x] All commits have Change-Id
- [ ] I have run tests with `make check`
- [x] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Comment by hfiguiere** [comment]
This is how it looks:

image

**Comment by mmeeks** [comment]
ooh - looks beautiful =) except that (somehow, one again) I see the classic toolbar which is an abomination =) how does that thing keep coming back like the bad penny =)

**Review by mmeeks — APPROVED** [review]
Looks really good to me Hubert =)

File: browser/html/framed.doc.html [hunk 1]
```diff
@@ -320,76 +356,122 @@ <h4>Example</h4>
...
+          <div class="vbox">
+            <button onclick="CollapseNotebookbar(true); return false;">Collapse Tabbed Toolbar</button>
+            <button onclick="CollapseNotebookbar(false); return false;">Extend Tabbed Toolbar</button>
+          </div>
+        </div>
+      </div>
+      <div class="vbox framed">
+        <h3>Send UNO Commands</h3>
+        <div class="hbox">
+          <button onclick="sendBoldUNOCommand(); return false;">Send Bold UNO Command (and example without args)</button>
```
Comment (timar) [inline browser/html/framed.doc.html:462]: *an* example without args

**Comment by timar** [comment]
> ooh - looks beautiful =) except that (somehow, one again) I see the classic toolbar which is an abomination =) how does that thing keep coming back like the bad penny =)

No worries the default is notebookbar. Probably Hubert pressed a few buttons before teking the screenshot, e.g. there are two save icons.

**Comment by hfiguiere** [comment]
> ooh - looks beautiful =) except that (somehow, one again) I see the classic toolbar which is an abomination =) how does that thing keep coming back like the bad penny =)

Probably because I was testing stuff and it persist into the local storage in the browser?

File: browser/html/framed.doc.html (hunk 1 above)
Comment (hfiguiere) [reply browser/html/framed.doc.html]: done

**Comment by hfiguiere** [comment]
I have added a fix for the "Show Hide save". Print did work thanks to #7589

**Review by mmeeks — APPROVED** [review]
Looks great to me thanks Hubert =)

**Pull request merged**